Jenkinsfile

```diff
@@ -44,7 +44,12 @@ pipeline {
 
 		stage('Tests') {
 			steps {
-				sh 'rm -f luacov.stats.* luacov.report.* testReport.xml cobertura.xml && ./test.sh --verbose --coverage --output junit > testReport.xml && ./lua_install/bin/luacov-cobertura -o cobertura.xml'
+				sh '''
```

Comment:
what does this change do?

Reply:
Rather than redirecting the test output, which includes print statements, it uses -Xoutput to pass in the name  of the output file. It also strips whitespace lines from the report.